hardhat.config.ts

```diff
@@ -1,22 +1,23 @@
-import type { HardhatUserConfig } from "hardhat/config";
+import type { HardhatUserConfig } from 'hardhat/config';
 
-import "@nomiclabs/hardhat-ethers";
-import "@nomiclabs/hardhat-waffle";
-import "@nomiclabs/hardhat-web3";
-import "@nomiclabs/hardhat-etherscan";
-import "@primitivefi/hardhat-dodoc";
-import "@typechain/hardhat";
-import "hardhat-deploy";
-import "hardhat-gas-reporter";
-import "solidity-coverage";
+import '@nomiclabs/hardhat-ethers';
+import '@nomiclabs/hardhat-waffle';
+import '@nomiclabs/hardhat-web3';
+import '@nomiclabs/hardhat-etherscan';
+import '@primitivefi/hardhat-dodoc';
+import '@typechain/hardhat';
```

Comment:
can you revert these changes from double quote to single quote?

Reply:
that was from me actually, are we enforcing double quotes on the sc repo?

Reply:
apparently not, because I see the prettierrc, ignore me, move along 🚃